modules/websearch/lib/websearch_webinterface.py

```diff
@@ -713,7 +778,7 @@ def answer(req, form):
                 if path[1] in ['', 'files', 'reviews', 'comments', 'usage',
                                'references', 'citations', 'holdings', 'edit',
                                'keywords', 'multiedit', 'merge', 'plots',
-                               'linkbacks', 'hepdata']:
+                               'linkbacks', 'hepdata', 'recommendations']:
```

Comment:
Its needed, because its in the 'WebInterfaceSearchInterfacePages' class and not in the 'WebInterfaceRecordRestrictedPages' class.